README.md

```diff
@@ -0,0 +1,73 @@
+# Metrics
+
+A simple, lightweight and extensible metrics gathering system.
+
+## Install
+
+```
+$ npm install @dxos/metrics
+```
+
+## Usage
+
+Metrics can either be submitted to the root object:
+
+```javascript
+import Metrics from '@dxos/metrics';
+
+Metrics.inc('work.started');
+```
+
+or to a nested object:
+
+```javascript
+import Metrics from '@dxos/metrics';
```

Comment:
I would use `metrics` in this case since is an instance and could be misunderstood with the { Metrics } class